src/demos/TypographyAlignment/TypographyAlignment.md

```diff
@@ -0,0 +1,128 @@
+```js { "props": { "data-example": "text" } }
+const { TextSize } = require('../../components/Text');
+
+<Block>
+  <Block>Text components of different sizes will share a common baseline:</Block>
+  <Block>
+    <Text>Unstyled</Text> <Text size={TextSize.XSMALL}>xsmall</Text> <Text size={TextSize.SMALL}>small</Text> <Text size={TextSize.MEDIUM_SUB}>mediumsub</Text> <Text size={TextSize.MEDIUM}>medium</Text> <Text size={TextSize.LARGE}>large</Text> <Text size={TextSize.XLARGE}>xlarge</Text> <Text size={TextSize.XXLARGE}>xxlarge</Text>
+  </Block>
+</Block>
```

Comment:
I don't think you don't need the wrapping `Block`. I seem to remember that Styleguidist is fine with multiple JSX nodes one after another, cause it wraps them in a container anyways.

Reply:
(I may be misremembering of course.)

Reply:
ya you have to end the example with a single JSX element or the example breaks :/